### ISSUE TITLE
Aztec Spear Animation

### ISSUE THREAD

**Issue by ZakJudges** [TheGameCreators/GameGuruRepo#3164]
Sometimes, after you have attacked with the Aztec Spear, you will see the spear being rotated back to its starting position. But it happens so fast that it looks like a bug. I have slowed the video down so you can see it better - see 0:05

https://user-images.githubusercontent.com/32545391/169514020-5bebf628-f6a5-490a-b8b4-764a598d681b.mp4

**Comment by LeeBamberTGC** [comment]
@bond1mjb Can you add a few more frames to this spear animation so the restoration of the spear rotates it out of view so it looks less like a glitch, and perhaps add some frames to rotate the spear at the start. The arrow head is point up when in idle, but directly down when the spear attack starts :)

Also, while you are on animation, can you lower the idle loop of the dagger animations so the hands are almost out of camera view and you see just the dagger (with a little bit of hand), similar to the axe idle pose.

**Comment by bond1mjb** [comment]
@LeeBamberTGC  Yes I can fix these issues on the spear.  It looks like the engine is doing is own keyframe interpolation to rotate the spear back to it's idle position, so I'll add a few more keyframes so it happen offscreen out of sight.

For the dagger, could you contact Gaby to provice the source 3ds Max files for those animations?  I will need them to do any modifications.

**Comment by LeeBamberTGC** [comment]
@bond1mjb I have asked for the dagger anim raw files :)  Will email them when I get 'em.

**Comment by bond1mjb** [comment]
@LeeBamberTGC  I've attached a new "attack2" animation.  There are now additional frames at the beginning  showing the spear rotating downward so the transition to the tip pointing down isn't instantaneous.   There are also additional frames at the end where it transitions back to the idle position offscreen.  Hopefully this fixes the glitchy look.

I've also attached a new "weapon.X" file that hopefully fixes the lighting seam on the spear
[spear fixes.zip](https://github.com/TheGameCreators/GameGuruRepo/files/8742208/spear.fixes.zip)
.
